src/Symfony/Component/HttpKernel/DependencyInjection/ServicesResetter.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\DependencyInjection;
+
+/**
+ * Resets provided services.
+ *
+ * @author Alexander M. Turek <me@derrabus.de>
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class ServicesResetter
+{
+    private $resettableServices;
+    private $resetMethods;
+
+    public function __construct(\Traversable $resettableServices, array $resetMethods)
+    {
+        $this->resettableServices = $resettableServices;
+        $this->resetMethods = $resetMethods;
+    }
+
+    public function resetServices()
```

Comment:
If the class is called `ServicesResetter`, shouldn't this method be called just `reset()` to follow [official Symfony naming conventions](https://symfony.com/doc/current/contributing/code/conventions.html)?

Reply:
why not, done :)